src/state/slices/opportunitiesSlice/opportunitiesSlice.test.ts

```diff
@@ -50,19 +51,56 @@ describe('opportunitiesSlice', () => {
           },
           type: DefiType.LiquidityPool,
         }
-        const expected = {
-          'eip155:1:0xMyContract': {
-            apy: '0.42',
-            assetId: foxEthLpAssetId,
-            provider: DefiProvider.FoxEthLP,
-            tvl: '424242',
-            type: DefiType.LiquidityPool,
-            underlyingAssetIds: [foxAssetId, ethAssetId],
-            underlyingAssetRatios: ['5000000000000000', '202200000000000000000'],
+        store.dispatch(opportunities.actions.upsertOpportunityMetadata(payload))
+        expect(store.getState().opportunities.lp.byId).toEqual(payload.byId)
+        expect(store.getState().opportunities.lp.ids).toEqual([mockLpContractOne])
+      })
+
+      it('merges prevState and payload', () => {
+        const insertPayloadOne: GetOpportunityMetadataOutput = {
+          byId: {
+            [mockLpContractOne]: {
+              // The LP token AssetId
+              assetId: foxEthLpAssetId,
+              provider: DefiProvider.FoxEthLP,
+              tvl: '424242',
+              apy: '0.42',
+              type: DefiType.LiquidityPool,
+              underlyingAssetIds: [foxAssetId, ethAssetId] as [AssetId, AssetId],
+              underlyingAssetRatios: ['0.005', '202.2'] as [string, string],
```

Comment:
is there a good reason we want to do these in human units over base units?

Reply:
Nice catch, that's a leftover from previous tests, this should be in base unit

Reply:
911b24632d438c4c0265f78e4cd1806358763166